lib/SIL/SILVerifier.cpp

```diff
@@ -2682,7 +2682,7 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     require(
         !isa<WitnessMethodInst>(branchArg) &&
```

Comment:
I think that's a bug @swiftix. A non-foreign `witness_method` should materialize to a function pointer value. Could you file a bug or radar and assign it to me?